WordPress/Classes/Models/Blog/Blog+WordPressComRestAPI.swift

```diff
@@ -1,7 +1,7 @@
 import WordPressData
 import WordPressKit
 
-public extension Blog {
+extension Blog {
```

Comment:
I was going to ask why this is no longer `public` but I'm pretty sure it's because the modifier is redundant given both members are declared as `public`.